common/app/conf/switches/CommercialSwitches.scala

```diff
@@ -298,6 +298,16 @@ trait CommercialSwitches {
     exposeClientSide = true
   )
 
+  val AdblockAsk = Switch(
+    Commercial,
+    "ab-adblock-ask",
+    "Places a contributions ask underneath the right-hand ad slot on articles.",
+    owners = group(Membership),
+    safeState = On,
```

Comment:
SafeState should probably be off?  🤔

Reply:
true, and it is currently switched off because of an issue with 'most viewed'